docs/markdown/DevGuide_BuildProcessor.md

```diff
@@ -0,0 +1,53 @@
+# Build your own Processor
+This guide will walk you through the creation of a new processor.
+
+## Necessary files and changes
```

Comment:
This is a bit confusing. First, you say that Inviwo-meta does all of these things and then you describe how to do them by hand. Stick to the automated pipeline and explain how to use the menu option instead.

The menu option is a one-size-fits-all solution and should be treated as such.